### ISSUE TITLE
The way you define raise is not intuitive

### ISSUE THREAD

**Issue by packetsss** [SirRender00/texasholdem#39]
When I raise 5 chips, the amount should be this pot's raised value + 5, not just 5 chips. It's causing many errors and I printed out player info's and pots.raised
![image](https://user-images.githubusercontent.com/76851428/157589133-6748b658-2272-480a-96f5-49c4cfaff474.png)

You can see in the image, player's actions do not map correctly to pot-raised amounts (when player 3 raised 1298 chips, one value in the list below should update to 1298 as the raised amounts) and I have no idea where to locate this bug.
Thanks for the help in advance!

**Comment by SirRender00** [comment]
Yeah so that's actually by design. That is `RAISE 50` can be read as raise _to_ 50. I do hear you though that it is way more intuitive to have the alternative. I'll add this in as a to do

**Comment by SirRender00** [comment]
A work around for now is that you can have a player raise the amount `game.chips_to_call(player_id) + raise_amount` where `raise_amount` behaves as you describe.

**Comment by packetsss** [comment]
Thank you for the reply! That's what I did using my agents, but as you can see in the screenshot above, pot.raised is not being calculated correctly and I have always been getting this:

![image](https://user-images.githubusercontent.com/76851428/157593591-f23252d7-c757-49f5-a55e-c08a85a3e9bf.png)

**Comment by SirRender00** [comment]
Looks to be a bug with split pots. What version are you on?

**Comment by packetsss** [comment]
The latest version

**Comment by SirRender00** [comment]
It would be also be helpful if you can export the history of the hand with `game.export_history` and post that here. It will be useful to replicate what happened and add it to the test suite.

**Comment by SirRender00** [comment]
Looks like that when someone went all-in.  It split the pot when it wasn't suppose to, probably has to do with the logic of when we split the pot here: https://github.com/SirRender00/texasholdem/blob/97e05f328413d8ead60a1d88c91d5ccf2ab559a9/texasholdem/game/game.py#L432

**Comment by SirRender00** [comment]
Ok I can see about getting a fix on this. But getting that history file if you can will be most helpful in verifying it.

**Comment by packetsss** [comment]
Not really sure haha, I added this in the Pot class
```
    def __repr__(self):
        return {
            "amount": self.amount,
            "raised": self.raised,
            "player_amounts": self.player_amounts,
            "player_amounts_without_remove": self.player_amounts_without_remove,
        }.__str__()
```

and the output looks like:
```
pots: [
    {
        "amount": 3000,
        "raised": 0,
        "player_amounts": {5: 0, 0: 0, 2: 0, 3: 0, 4: 0},
        "player_amounts_without_remove": {
            0: 500,
            1: 500,
            2: 500,
            3: 500,
            4: 2622,
            5: 500,
        },
    },
    {
        "amount": 0,
        "raised": 0,
        "player_amounts": {4: 0, 0: 0, 2: 0, 3: 0},
        "player_amounts_without_remove": {4: 2122, 0: 0, 1: 0, 2: 0, 3: 0},
    },
    {
        "amount": 0,
        "raised": 0,
        "player_amounts": {4: 0, 2: 0, 3: 0},
        "player_amounts_without_remove": {4: 2122, 1: 2122, 2: 0, 3: 0},
    },
    {
        "amount": 0,
        "raised": 0,
        "player_amounts": {4: 0, 3: 0},
        "player_amounts_without_remove": {4: 2122, 1: 2122, 3: 0},
    },
    {
        "amount": 2122,
        "raised": 0,
        "player_amounts": {4: 0},
        "player_amounts_without_remove": {4: 1061, 1: 1061},
    },
]
```
I also added this player_amounts_without_remove that records player's bets in the entire game without resetting it. 
**Seems like a lot of empty pots are added.**

**Comment by packetsss** [comment]
Here are some of the historys...
[Uploading history.zip…]()

**Comment by SirRender00** [comment]
Sorry which of those files display abnormal behavior?

**Comment by packetsss** [comment]
> Sorry which of those files display abnormal behavior?

I don't think there're any more😂, just sometimes empty pots are added

**Comment by SirRender00** [comment]
Got it haha... if you find a case... just attach it here. Hopefully there's nothing huge then that's blocking you. As a very yucky solution at each step you can run

`
pots_to_remove = []
for i, pot in enumerate(game.pots, 0):
	if pot.get_total_amount() == 0:
		players_to_decrement = {}
		for later_pot in game.pots[(i+1):]:
			players_to_decrement.union(later_pot.player_amounts.keys())
		for id in players_to_decrement:
			game.players[id].last_pot -= 1
		pots_to_remove.append(i)

game.pots = [pot
		      for i, pot in enumerate(game.pots, 0) 
		      if i not in pots_to_remove]
`

**Comment by SirRender00** [comment]
Once we find a case where there's empty pots we can work on this

**Comment by SirRender00** [comment]
- [x] Rename value to total and deprecate it for 1.0
- [] 1.0 Do the change and fix all the calls
